memchain/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![feature(maybe_uninit)]
 
 mod block;
+#[cfg(features = "ffi")]
```

Comment:
why?

Reply:
because I don't want to pollute statically linked libs with useless global symbols if they're not needed